packages/server/modules/shared/services/auth.ts

```diff
@@ -106,7 +114,27 @@ export const authorizeResolverFactory =
       : null
 
     if (!userAclRole) {
-      throw new ForbiddenError('You are not authorized to access this resource.')
+      // TODO: Could be more optimized (caching?) but we're moving away from this towards
```

Comment:
Fixed up authorizeResolver to take into account implicit workspace roles. Ofc we need new policies for mutations anyway, because the frontend needs them for checks